### ISSUE TITLE
Switch to help wanted and good first issue in all projects

### ISSUE THREAD

**Issue by rprouse** [nunit/governance#31]
GitHub is now promoting these specific labels, so we should rename our `up-for-grabs` and `easyfix` to match so GitHub will send people to the repo. I have switched the NUnit and NUnit Console repos, the others need to be done too.

![image](https://user-images.githubusercontent.com/493828/31504693-09a19edc-af41-11e7-92e9-e1f31fba9e74.png)

**Comment by OsirisTerje** [comment]
Fixed NUnit3TestAdapter and TestGenerator

**Comment by jnm2** [comment]
Side note. I pinged GitHub support to see if they'd work with easy-fix and up-for-grabs. I also asked about dotfile configuration if for some reason they don't want to globally recognize them.

Reply:

> Thanks for writing us today! At this time, we don't have an option to use customized labels, like your existing up-for-grabs label, interchangeable with our "help wanted" and "good first issue" labels. However, we are collecting all feedback on this new feature, and we're definitely paying close attention to what users like yourself think of the updates.
> 
> I've made a note of your case for our ongoing conversations about how to improve GitHub. Sorry if that wasn't what you were hoping to hear, but please let us know if you have any other questions.

**Comment by jnm2** [comment]
We typically remove `up-for-grabs` as soon as the issue is assigned to a contributor but not `easyfix`. Now though, if GitHub is going make `good first issue` issues specially visible to first-time contributors, do we remove that one too? Or should we rename it back and start a new `good first issue` tag?

**Comment by jnm2** [comment]
When we decide, we should also send a PR to up-for-grabs.net.

**Comment by rprouse** [comment]
> if GitHub is going make good first issue issues specially visible to first-time contributors, do we remove that one too?

I don't think we need to do that, the user should be able to see that it is taken.
